### PR TITLE
Add Class 10

### PR DESCRIPTION
**Pull request by ThomasTheDane**

### CODE REVIEW

File: lab1/README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+- List out all pages on various topics that have pages
+- Read any submitted pages and their content
+- Allow users to submit new pages on topics
+- Allow users to submit edits to pages
+- Documentation for each major component
+- Not look terrible
+- One other feature you'd like to implement (Could be as simple as a random page or something more serious like searching, but we are urging creativity)
+
+If you want to make the wiki have a special topic area (like harry potter or some game) you are encouraged to do so. 
+Brownie points will be awarded for awesome looking websites whoose themes match the topic area, though have all the functionallity should come before making it pretty.
```
Comment (EvanDorsky) [inline lab1/README.md]: Change to:

> Brownie points will be awarded for awesome looking websites whose themes match the topic area, though having all the functionallity should come before making it pretty.

**Comment by ThomasTheDane** [comment]
I just pushed the changes we discussed over dinner. Look through the two readmes and let me know what you think.

File: lab1/README.md [hunk 2]
```diff
@@ -1 +1,38 @@
-#Lab 1
+#Lab 1: WikiMVC
+
+##Intro
+This lab will have 3 components: 
+- Choosing a client-side Javascript framework and learning it 
+- Constructing a wiki clone with certain functionality in that framework
+- Preparing a presentation to the class about what your chosen framework
```
Comment (benkahle) [inline lab1/README.md:7]: about your chosen framework

File: lab1/README.md [hunk 3]
```diff
@@ -1 +1,38 @@
-#Lab 1
+#Lab 1: WikiMVC
+
+##Intro
+This lab will have 3 components: 
+- Choosing a client-side Javascript framework and learning it 
+- Constructing a wiki clone with certain functionality in that framework
```
Comment (benkahle) [inline lab1/README.md:6]: Using that framework to construct a wiki clone with specified functionality

File: lab1/README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+- Preparing a presentation to the class about what your chosen framework
+
+The lab will be completed in pairs. You must collaborate on github repo separate from either of your olinjs repos.
```
Comment (benkahle) [inline lab1/README.md:9]: a new github repository

File: lab1/README.md [hunk 5]
```diff
@@ -1 +1,38 @@
...
+
+##Choosing a Framework
+Each pair will be choosing a client side framework. Angular, Backbone (potentially with Marionette), Ember, React, Flight, Knockout, an Riot are all valid options but there are many other options.
```
Comment (benkahle) [inline lab1/README.md:12]: and Riot.

but -> "and" there are many others.

File: lab1/README.md [hunk 6]
```diff
@@ -1 +1,38 @@
...
+If you are unsure of a framework qualifies just check with us and we should be able to tell you very quickly. 
+
+##Constructing a Wiki
+You must construct a wikipedia-like clone using your chosen framework. You should program the app using the conventions, styles, and functionality of your chosen framework, show off what it can do. It should have the following functionality:
+- List out all pages on various topics that have pages
```
Comment (benkahle) [inline lab1/README.md:17]: A list of all available pages

File: lab1/README.md [hunk 7]
```diff
@@ -1 +1,38 @@
...
+- Read any submitted pages and their content
```
Comment (benkahle) [inline lab1/README.md:18]: Move this first
Topic pages with title and content.

File: lab1/README.md [hunk 8]
```diff
@@ -1 +1,38 @@
...
+If you want to make the wiki have a special topic area (like harry potter or some game) you are encouraged to do so. 
+Brownie points will be awarded for awesome looking websites whose themes match the topic area, though having all the functionallity should come before making it pretty.
+
+##Framework Presentation
+You and your partner will get together with the other groups who are doing the same framework and create a 8-10 minute presentation about the framework. 
+The goal of the presentation is NOT to teach the class to use the framework. 
+Rather the presentation should contain the following pieces:
+- Basic overview and introduction to the framework
+- Any unique features that you believe are worth showing off
+- What are some advantages of the framework
```
Comment (benkahle) [inline lab1/README.md:35]: add: What did you enjoy about using the framework

File: lab1/README.md [hunk 9]
```diff
@@ -1 +1,38 @@
...
+- What are some disadvantages / difficulties of the framework
```
Comment (benkahle) [inline lab1/README.md:36]: add: What specific issues did you have using the framework

File: lab1/README.md [hunk 10]
```diff
@@ -1 +1,38 @@
...
+- What type of project would the framework be suited or not suited for
```
Comment (benkahle) [inline lab1/README.md:37]: "Is there a specific type of project that the framework would work or not work well for?"

File: lab1/README.md [hunk 11]
```diff
@@ -1 +1,38 @@
...
+- Anything else you believe the class should know about the framework
```
Comment (benkahle) [inline lab1/README.md:38]: Add a point on availability of documentation, community support and activity, direction of growth, current popularity, etc.

File: classes/class10/README.md [hunk 12]
```diff
@@ -1 +1,98 @@
...
+Quote from the top of their web page: "A framework for creating **ambitious** web applications"
+
+###Angular
+Angular is developed by Google and has as its chief aim to make html what it would be if it was created today.
+It has several interesting qualities such as html directives.
+A primary focus of angular is to build for testability and it has functionality to help enable exactly that. 
+
+###React
+Developed by Facebook and focuses on the view part of MVC.
+Mission statement is to allow users to large applications with data that changes over time.
```
Comment (benkahle) [inline classes/class10/README.md:79]: to create large applications

**Pull request merged**